esphomeyaml/components/sensor/dht.rst

```diff
@@ -62,7 +64,7 @@ Configuration variables:
   - All other options from :ref:`Sensor <config-sensor>` and :ref:`MQTT Component <config-mqtt-component>`.
 
 - **model** (*Optional*, int): Manually specify the DHT model, can be
-  one of ``AUTO_DETECT``, ``DHT11``, ``DHT22``, ``AM2302``, ``RHT03``
+  one of ``AUTO_DETECT``, ``DHT11``, ``DHT22``, ``AM2302``, ``RHT03``, ``SI7021``
```

Comment:
Auto detection won't work for this chip. We should make that clear in the docs.

Reply:
Does it work for RHT03 and AM2302 ? I couldn't see it in the code.

Reply:
Well the RHT03 and AM2302 are just aliases for the DHT22 - so the option is pretty much just there as a placebo :)